go/teams/handler.go

```diff
@@ -117,16 +118,24 @@ func handleSBSSingle(ctx context.Context, g *libkb.GlobalContext, teamID keybase
 		}
 		assertion := string(invite.Name) + "@" + ityp
 
-		res := g.Resolver.ResolveWithBody(assertion)
-		if res.GetError() != nil {
-			return res.GetError()
+		arg := keybase1.Identify2Arg{
```

Comment:
BTW, you can also chain the uid assertion together with the SBS assertion above, that might be better than checking it after the fact.